packages/reports/tests/acceptance/schedule-modal-test.js

```diff
@@ -200,9 +200,11 @@ module('Acceptance | Navi Report Schedule Modal', function (hooks) {
       'Recipients field is populated by existing delivery rule'
     );
 
-    assert
-      .dom('.schedule__modal-format-trigger .ember-power-select-selected-item')
-      .hasText('csv', 'Format field is populated by existing delivery rule');
+    assert.equal(
+      $('.schedule__modal-format-trigger .ember-power-select-selected-item').text().trim(),
+      { type: 'csv' },
+      'Format field is populated by existing delivery rule'
+    );
```

Comment:
aren't we converting from `{ type: 'csv' }` to just `'csv'`? shouldn't the old assert still work here?